bats/fb-content-katello.bats

```diff
@@ -245,30 +261,20 @@ setup() {
 }
 
 @test "publish content view" {
+  tSkipIfNoPulp2 "Puppet content"
```

Comment:
publish and promote should still work with pulp3 only?

Reply:
They should, but this publish and promote were targeted as follow ons to adding the Puppet content so they are not really testing anything new.

Reply:
oooh, right. should the test name then be adjusted to "$action content view with puppet content" or something?